snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/validation/SnomedQueryValidationRuleEvaluator.java

```diff
@@ -143,6 +130,7 @@ public String type() {
 		@JsonProperty private Boolean active;
 		@JsonProperty private String effectiveTime;
 		@JsonProperty private String module;
+		@JsonProperty private Boolean filterByWorkingModule;
```

Comment:
It is possible to define a default value for a JsonProperty (by using `defaultValue`). I would suggest setting this value to false by default, so then you don't need to change so many rule declarations. This way we will have three ideal scenarios (and a not-quite-ideal fourth):
- there is neither `module` nor `filterByWorkingModule` value specified in the rule implementation --> do not apply any module filtering
- there is no `module` value but `filterByWorkingModule = true` --> apply module filtering based on the default code system modules
- there is a `module` value (e.g. an ECL expression) and `filterByWorkingModule` is not present (== false) --> filter by module using the expression in the property `module`
- there is a `module` value and `filterByWorkingModule = true` --> this is a contradiction but we can still say to use the value in the `module` parameter and filter by that, assuming that must be something more specific provided on purpose

Reply:
Done